private/buf/cmd/buf/command/push/push.go

```diff
@@ -371,16 +402,16 @@ func validateCreateFlags(flags *flags) error {
 }
 
 func combineLabelLikeFlags(flags *flags) []string {
-	return slicesext.ToUniqueSorted(
-		append(
-			flags.Labels,
-			append(
-				flags.Tags,
-				flags.Draft,
-				flags.Branch,
-			)...,
-		),
-	)
+	labels := make([]string, 0, len(flags.Labels)+len(flags.Tags))
```

Comment:
Why not continue to use previous style, `labels := append(flag.Labels, flags.Tags...)`?

Reply:
`append` [can mutate the underlying array of its first argument](https://trstringer.com/golang-append/) (`flag.Label`). It shouldn't matter in this case because we're not overwriting indexes, but in general I avoid using `append` to copy a slice I don't own. Happy to change this back if you think it's a non-issue.

Reply:
Well every day's a school day I guess, I don't know how I've gone this long without running into some bug based on this. Sounds good. A slight nit, I might clean it up to `append(slicesext.Copy(flags.Labels), flags.Tags...)` but doesn't matter much.

Reply:
I cleaned it up using `slicesext.Copy`

Reply:
@akshayjshah on my case about how could I be so naive